crates/bevy_core_pipeline/src/bloom/mod.rs

```diff
@@ -479,15 +543,22 @@ impl FromWorld for BloomPipelines {
 
         let upsampling_pipeline = pipeline_cache.queue_render_pipeline(RenderPipelineDescriptor {
             label: Some("bloom_upsampling_pipeline".into()),
-            layout: Some(vec![upsampling_bind_group_layout.clone()]),
+            layout: Some(vec![main_bind_group_layout.clone()]),
             vertex: fullscreen_shader_vertex_state(),
             fragment: Some(FragmentState {
                 shader: BLOOM_SHADER_HANDLE.typed::<Shader>(),
                 shader_defs: vec![],
                 entry_point: "upsample".into(),
                 targets: vec![Some(ColorTargetState {
-                    format: ViewTarget::TEXTURE_FORMAT_HDR,
-                    blend: None,
+                    format: TextureFormat::Rg11b10Float,
```

Comment:
Performance, theoretically. We don't need the alpha channel, and this way we save some memory usage / bandwidth. Last slide of the call of duty presentation mentions it, as do a few other places. 

That said, I haven't figured out how to get NSight to profile render passes, so I haven't been able to validate it.

Reply:
Instead of removing it, we can take the advantage of the alpha channel and blend based on fragment output alpha. That way we can elliminate texture_b and save memory that way. I think that might be the more intuitive solution at least